content/en/docs/azure/authentication-oidc.md

```diff
@@ -0,0 +1,209 @@
+
+    ```
+    kfctl apply -V -f ${CONFIG_URI}
+    ```
+
+4. Check the resources deployed correctly in namespace `kubeflow`
+
+    ```
+    kubectl get all -n kubeflow
+    ```
+
+## Expose Kubeflow securely over HTTPS
+
+1. Update Istio Gateway to expose port 443 with HTTPS and make port 80 redirect to 443:
+
+    ```
+    kubectl edit -n kubeflow gateways.networking.istio.io kubeflow-gateway
+    ```
+
+    The Gateway spec should look like the following:
+
+    {{< highlight yaml >}}
+    apiVersion: networking.istio.io/v1alpha3
+    kind: Gateway
+    metadata:
+    name: kubeflow-gateway
+    namespace: kubeflow
+    spec:
+    selector:
+        istio: ingressgateway
+    servers:
+    - hosts:
+        - '*'
+        port:
+        name: http
+        number: 80
+        protocol: HTTP
+              # Upgrade HTTP to HTTPS
+        tls:
+        httpsRedirect: true
+    - hosts:
+        - '*'
+        port:
+        name: https
+        number: 443
+        protocol: HTTPS
+        tls:
+        mode: SIMPLE
+        privateKey: /etc/istio/ingressgateway-certs/tls.key
+        serverCertificate: /etc/istio/ingressgateway-certs/tls.crt
+    {{< /highlight >}}
+
+1. Expose Kubeflow with a loadBalancer service:
+
+    To expose Kubeflow with a LoadBalancer service, just change the type of the `istio-ingressgateway` service to `LoadBalancer`.
+
+    ```
+    kubectl patch service -n istio-system istio-ingressgateway -p '{"spec": {"type": "LoadBalancer"}}'
+    ```
+
+   After that, get the LoadBalancer's IP or Hostname from its status and create the necessary certificate.
+
+    ```
+    kubectl get svc -n istio-system istio-ingressgateway -o jsonpath='{.status.loadBalancer.ingress[0]}'
+    ```
+
+1. Create the Certificate with cert-manager:
+
+    Create a new `certficate.yaml` with below yaml to create a self-signed Certificate with the cert-manager. For production environment please use appropriate trusted CA Certificate.
+
+    {{< highlight yaml >}}
+    apiVersion: cert-manager.io/v1alpha2
+    kind: Certificate
+    metadata:
+    name: istio-ingressgateway-certs
+    namespace: istio-system
+    spec:
+    commonName: istio-ingressgateway.istio-system.svc
+        # Use ipAddresses if your LoadBalancer issues an IP
+    ipAddresses:
+    - <LoadBalancer IP>
+        # Use dnsNames if your LoadBalancer issues a hostname
+    dnsNames:
+    - <LoadBalancer HostName>
+    isCA: true
+    issuerRef:
+        kind: ClusterIssuer
+        name: kubeflow-self-signing-issuer
+    secretName: istio-ingressgateway-certs
+    {{< /highlight >}}
+
+    Apply `certificate.yaml` in `istio-system` namespace
+
+    ```
+    kubectl apply -f certificate.yaml -n istio-system
+    ```
+
+    After applying the above Certificate, cert-manager will generate the TLS certificate inside the istio-ingressgateway-certs secrets. The istio-ingressgateway-certs secret is mounted on the istio-ingressgateway deployment and used to serve HTTPS.
```

Comment:
done

Reply:
One more thing here, since you're setting `secretName` to `istio-ingressgateway-certs` in that YAML, let's use backticks (it helps new DevOps users visually):

```suggestion
    After applying the above Certificate, cert-manager will generate the TLS certificate inside the `istio-ingressgateway-certs` secrets. The `istio-ingressgateway-certs` secret is mounted on the `istio-ingressgateway` deployment and used to serve HTTPS.
```